clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2489,6 +2492,7 @@ def GlobalOp : CIR_Op<"global",
        ($tls_model^)?
        (`dsolocal` $dsolocal^)?
        (`addrspace` `(` custom<GlobalOpAddrSpace>($addr_space)^ `)`)?
+       (`actual_addrspace` `(` custom<GlobalOpAddrSpace>($actual_addr_space)^ `)`)?
```

Comment:
Maybe we don't need this, but if we do, `actual_addrspace` -> `gpu_addrspace` would probably be more descriptive?